backend/src/main/scala/cromwell/backend/validation/RuntimeAttributesValidation.scala

```diff
@@ -68,6 +68,13 @@ object RuntimeAttributesValidation {
     }
   }
 
+  def validateBigDecimal(value: WomValue): ErrorOr[BigDecimal] = {
+    WomIntegerType.coerceRawValue(value) match {
```

Comment:
It is definitely exhaustive match, but you can do it with `Success` and `Failure` and it will be nicer 
Also, you can import `Success` and `Failure` and remove path declaration. It also will be much appreciated. 
like 
```import scala.util.{Try, Success, Failure}```

Reply:
Oh, and also there is an incorrect string. Thanks, I did not noticed it before.

Reply:
Fixed.